### PR TITLE
Extend get_alert() with another 3 return values

### PR DESCRIPTION
**Pull request by bachradsusi**
The new signature for get_alert is:

    get_alert(s: local_id) -> ssiasa(ssssbb)sss

Last 3 values are:

* `first_seen_date(s)`: when the alert was seen for the first time, iso8601 format is used - `%Y-%m-%dT%H:%M:%SZ`
* `last_seen_date(s)`: when the alert was seen for the last time, iso8601 format is used - `%Y-%m-%dT%H:%M:%SZ`
* `level(s)`: "green", "yellow" or "red"

Fixes: https://github.com/fedora-selinux/setroubleshoot/issues/20 https://github.com/fedora-selinux/setroubleshoot/issues/19

Note: Based on discussion with @dperpeet I decided to break get_alert() API, and add another 3 values to the end of return values.

### CODE REVIEW

**Comment by bachradsusi** [comment]
There'll be another pull request with get_alert_info() which will provide complete structure which is used in setroubleshoot - https://github.com/fedora-selinux/setroubleshoot/blob/master/framework/src/setroubleshoot/signature.py#L246

**Comment by bachradsusi** [comment]
`setroubleshoot-3.3.5-0.1` builds with these fixes will be available for f23, f24 and rawhide  in https://copr.fedorainfracloud.org/coprs/plautrba/setroubleshoot/ soon

File: framework/DBUS.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ see get_all_alerts()
 
 ***
 
-##### get_alert(s: local_id) -> ssiasa(ssssbb)
+##### get_alert(s: local_id) -> ssiasa(ssssbb)sss
```
Comment (dperpeet) [inline framework/DBUS.md]: the array signature doesn't have the extra `i` for the priority
I think:
`##### get_alert(s: local_id) -> ssiasa(ssssbbi)sss`

**Comment by bachradsusi** [comment]
Thanks, I've fixed the signature and merged it.  https://github.com/fedora-selinux/setroubleshoot/commit/5b4f69e696b989378f7d1a42959bae3c695321e6  https://github.com/fedora-selinux/setroubleshoot/commit/900166830aea54cc3534fd638e8d3637bf32eab4 https://github.com/fedora-selinux/setroubleshoot/commit/42edc1dcae82da6dd79f4437631eb0f0ebb11bbd

**Pull request closed without merging**